src/injectors/github-injector.ts

```diff
@@ -147,6 +154,20 @@ abstract class ButtonInjectorBase implements ButtonInjector {
     protected adjustButton(a: HTMLAnchorElement) {
         // do nothing
     }
+
+    protected renderEditButton(url: string, openAsPopup: boolean): HTMLElement {
+        const a = document.createElement('a');
+        a.id = Gitpodify.EDIT_BTN_ID;
+        a.title = "Edit this file in Gitpod";
+        a.text = "Open in Gitpod";
+        a.href = url;
+        a.target = "_blank";
```

Comment:
I know this has already been merged, but maybe we should make it open in the current tab by default, since that's what `Open in github.dev` does. We could maybe have an option for that as well in the extension settings.

cc @gtsiolis @Siddhant-K-code

Reply:
@filiptronicek You're right! Could you open a follow-up issue or PR for this? 🏓

Reply:
I think, currently we open Gitpod Workspace in new tab. So followed that convention. Because for User: In `github.dev`it is github to github but in `open in gitpod` it is GitHub to Gitpod.

Reply:
It's ok—Let's leave this as is. Thanks again @Siddhant-K-code for contributing with this feature!